src/main/java/org/snomed/snowstorm/core/data/domain/Relationship.java

```diff
@@ -129,7 +136,17 @@ public Relationship(String id, Integer effectiveTime, boolean active, String mod
 		this.active = active;
 		setModuleId(moduleId);
 		this.sourceId = sourceId;
-		this.destinationId = destinationId;
+
+		if (destinationId != null) {
+			if (destinationId.startsWith("#") || destinationId.startsWith("\"")) {
```

Comment:
I'm not sure about this but we may be able to rid of it later.

Reply:
We should at least name the parameter as destinationIdOrValue until we know what it is.